src/braket/pennylane_plugin/ops.py

```diff
@@ -248,6 +305,9 @@ def _matrix(cls):
             )
         )
 
+    def adjoint(self):
+        return ECR(wires=self.wires)
+
 
 class PSWAP(Operation):

```

Comment:
Should this also have a generator?

Reply:
@licedric said that PSWAP won't have a generator since it can't be written in that form. All the classes have been added as per Cedric's suggestions.